Keyboards/InterfaceConstants.swift

```diff
@@ -1,8 +1,22 @@
-//
-//  Scribe
-//
-//  Constants for Scribe keyboard interfaces.
-//
+/**
+ * Constants for the Scribe keyboard interfaces
+ *
+ *
+ * Copyright (C) 2023 Scribe
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
```

Comment:
Hey!! :wave: 

Just wondering about something here.. 
How would we plan on maintaining this license header in all the relevant files? Not sure if some automation was used to make the changes in this PR, but could there be something like [.editorconfig](https://editorconfig.org/) that we could use to generate these (especially if we'd like to tweak the text in the future)?

CC: @andrewtavis

Reply:
Any level of automation would be appreciated for this. I believe that @Jag-Marcel was going through individually as this was also a task of better understanding the codebase. The file headers also have a description of the file, which we thought was important. Could you send along an example of how the license could be automated with this, @wkyoshida?

Reply:
It seems like this might be possible with Xcode perhaps ([ref](https://stackoverflow.com/questions/69420028/incomplete-file-copyright-headers-in-swift-package-with-xcode-12-13))? I don't have Xcode to be able to validate if this works though..

Reply:
This seems to be a bit more for changing the default for when the file is made, but maybe I'm not reading it clearly 🤔 Would be good to include though! I'd say it's a different issue though.